msgs/include/simple_msgs/image.hpp

```diff
@@ -152,30 +152,30 @@ class Image : public GenericMessage {
     return std::make_shared<flatbuffers::DetachedBuffer>(builder->Release());
   }
 
-  std::array<double, 3> getResolution() const { return {{resX_, resY_, resZ_}}; }
-  std::array<int, 3> getImageDimensions() const { return {{width_, height_, depth_}}; }
+  std::array<double, 3> getSpacing() const { return {{spacing_x_, spacing_y_, spacing_z_}}; }
+  std::array<uint32_t, 3> getImageDimensions() const { return {{width_, height_, depth_}}; }
   const T* getImageData() const { return data_.getData(); }
-  int getImageSize() const { return data_size_; }
+  uint64_t getImageSize() const { return data_size_; }
   const Header& getHeader() const { return header_; }
   Header& getHeader() { return header_; }
   const Pose& getImageOrigin() const { return origin_; }
   Pose& getImageOrigin() { return origin_; }
   std::string getImageEncoding() const { return encoding_; }
-  int getNumChannels() const { return num_channels_; }
+  unsigned int getNumChannels() const { return num_channels_; }
```

Comment:
this should be uint32_t for consistency

Reply:
you mean unsigned int?

I changed num_channel to that

 ```unsigned short num_channels_{1};```

I guess that for the number of channels we can use something very tiny.

Reply:
but then why I didn't use uint16_t? :D